src/metrocs/HelloAnyoneTest.java

```diff
@@ -1,5 +1,4 @@
 package metrocs;
-import org.junit.jupiter.api.Test;
```

Comment:
Note that the "error" is pointing out a shortcoming that needs to be addressed.
We need to consider why the error exists and what it is telling us.
What's actually needed here--to "fix" the "Checkstyle error"--is for there to be an actual test in the file.
Removing the import only "hides" the fact that the test class has no actual tests.
(This is similar to a "TODO:" error which is again indicating that something needs to be added rather than taken away.)